modules/ROOT/pages/index.adoc

```diff
@@ -28,6 +31,29 @@ Some capabilities of Anypoint Runtime Fabric include:
 * Automated application fail-over.
 * Application management with Anypoint Runtime Manager.
 
+The Anypoint control plane features, Anypoint CLI, and Mule Maven plugin are available universally across all partners. At runtime plane only Helm based Runtime Fabric management and features are supported for VMWare Tanzu, Alibaba ACK and Rancher. The feature supportability matrix across platforms is as follows:
```

Comment:
If we are talking about THE (our) Mule Maven Plugin, "Plugin" should be capitalized.